linux_explorer.py

```diff
@@ -125,6 +125,16 @@ def find_by_type(type):
 
     return jsonify(otx.get_indicator_details_full(find_by_type(type), indicator)), 200
 
+@app.route('/malshare/<string:type>/<string:indicator>')
+def malshare_report(type, indicator):
+    ''' MalShare repository.
+        Supported indicator types: "domain", "hostname", "URL", "FileHash-MD5", "FileHash-SHA1", "FileHash-SHA256", '''
+
+    response = requests.get('https://malshare.com/api.php?api_key=%s&action=details&hash=%s' % ( config.MALSHARE_APIKEY, indicator ), headers={'Accept-Encoding': 'gzip, deflate', 'User-Agent': 'gzip,  Linux Expl0rer'})
```

Comment:
can you handle no api key similar to [otx](https://github.com/intezer/linux-explorer/pull/1/files#diff-24c478777ae4b5954679a8c3125a85f1R121)?